docker/start.py

```diff
@@ -205,41 +205,53 @@ def main(args, environ):
             config_dir, config_path, environ, ownership
         )
 
-    if mode is not None:
-        error("Unknown execution mode '%s'" % (mode,))
+    if mode == "run":
```

Comment:
I suggest you keep this as an error check, to save having a massive `if` statement:

```suggestion
    if mode != "run":
        error("Unknown execution mode '%s'" % (mode,))
```